Add TRIGGERcmd integration documentation (#33575)
home-assistant/home-assistant.github.io: source/_integrations/triggercmd.markdown

---
title: TRIGGERcmd
description: How to integrate TRIGGERcmd with Home Assistant.
ha_category:
  - Switch
  - Automation
ha_release: "2024.10"
ha_iot_class: Cloud Polling
ha_config_flow: true
ha_codeowners:
  - '@rvmey'
ha_domain: triggercmd
ha_platforms:
  - switch
ha_zeroconf: false
ha_integration_type: integration
---

The TRIGGERcmd {% term integration %} allows you to run commands on computers via [TRIGGERcmd](https://triggercmd.com/) with Home Assistant.

## Prerequisites

To use TRIGGERcmd, you need the following:

- A [TRIGGERcmd account](https://www.triggercmd.com/user/auth/signup)
- A computer with the TRIGGERcmd agent running on it
- For instructions on installing and using TRIGGERcmd itself, refer to the [TRIGGERcmd Quick Start Guide](https://docs.triggercmd.com/#/./QuickStart).
- After adding the TRIGGERcmd integration, you will be prompted for your **user token**. This can be found at the bottom of your TRIGGERcmd profile page or the instructions page.
{% include integrations/config_flow.md %}

## Using TRIGGERcmd with Home Assistant

> All of your TRIGGERcmd commands should show up as **switch** devices. You can change the rooms of your devices, or press **finish** to leave them without rooms.

Reviewer: ```suggestion
All of your TRIGGERcmd commands should show up as **switch** devices. You can change the rooms of your devices, or select **Finish** to leave them without rooms.
```

---

> title: TRIGGERcmd Integration

Reviewer: ```suggestion
title: TRIGGERcmd
```

description: How to integrate TRIGGERcmd with Home Assistant.
ha_category:
  - Switch
  - Automation

> ha_release: '2024.6'

Reviewer 2: ```suggestion
ha_release: "2024.10"
```

ha_iot_class: Cloud Polling
ha_config_flow: true
ha_codeowners:
  - '@rvmey'
ha_domain: triggercmd
ha_platforms:
  - switch
ha_zeroconf: false
ha_integration_type: integration
---

> This integration allows you to run commands on computers via [TRIGGERcmd](https://triggercmd.com/) with Home Assistant.

Reviewer: ```suggestion
The TRIGGERcmd {% term integration %} allows you to run commands on computers via [TRIGGERcmd](https://triggercmd.com/) with Home Assistant.
```

> {% include integrations/config_flow.md %}

Reviewer: ```suggestion
```

> Instructions on how to install and use TRIGGERcmd itself can be found [here](https://docs.triggercmd.com/#/./QuickStart).

Reviewer: ```suggestion
```

{% include integrations/config_flow.md %}

## Prerequisites

> To use TRIGGERcmd, you need the following:

Reviewer: ```suggestion

To use TRIGGERcmd, you need the following:
```

- A computer with the TRIGGERcmd agent running on it

> - For instructions on installing and using TRIGGERcmd itself, refer to the [TRIGGERcmd Quick Start Guide](https://docs.triggercmd.com/#/./QuickStart).

Reviewer: ```suggestion
- For instructions on installing and using TRIGGERcmd itself, refer to the [TRIGGERcmd Quick Start Guide](https://docs.triggercmd.com/#/./QuickStart).
- After adding the TRIGGERcmd integration, you will be prompted for your **user token**. This can be found at the bottom of your TRIGGERcmd profile page or the instructions page.
```

- A computer with the TRIGGERcmd agent running on it

> 

Reviewer: ```suggestion
- For instructions on installing and using TRIGGERcmd itself, refer to the [TRIGGERcmd Quick Start Guide](https://docs.triggercmd.com/#/./QuickStart).
```

> - For instructions on installing and using TRIGGERcmd itself, refer to the [TRIGGERcmd Quick Start Guide](https://docs.triggercmd.com/#/./QuickStart).

Reviewer: ```suggestion
- For instructions on installing and using TRIGGERcmd itself, refer to the [TRIGGERcmd Quick Start Guide](https://docs.triggercmd.com/#/./QuickStart).

```

> 

Reviewer: ```suggestion
{% include integrations/config_flow.md %}

```

## Using TRIGGERcmd with Home Assistant

> Upon adding the TRIGGERcmd integration, you will be prompted for your **user token,** this can be found at the bottom of your TRIGGERcmd profile page or the instructions page.

Reviewer: ```suggestion
```

> 

Reviewer: ```suggestion
```

---
title: TRIGGERcmd
description: How to integrate TRIGGERcmd with Home Assistant.
ha_category:
  - Switch
  - Automation
ha_release: "2024.10"
ha_iot_class: Cloud Polling
ha_config_flow: true
ha_codeowners:
  - '@rvmey'
ha_domain: triggercmd
ha_platforms:
  - switch
ha_zeroconf: false
ha_integration_type: integration
---

The TRIGGERcmd {% term integration %} allows you to run commands on computers via [TRIGGERcmd](https://triggercmd.com/) with Home Assistant.

> 

Reviewer: ```suggestion
```
